CBLClient/Apps/CBLTestServer-C/build_scripts/build_cross.py

```diff
@@ -146,7 +146,10 @@ def check_sysroot(name: str):
     os.chdir(DOWNLOAD_DIR)
 
     print(f"==== Downloading Couchbase Lite C [{args.os}] {args.version}-{args.bld_num} ====")
-    zip_filename=f'couchbase-lite-c-{args.edition}-{args.version}-{args.bld_num}-{args.os}.tar.gz'
+    os_x86_64_list = ["debian9-x86_64", "debian10-x86_64", "ubuntu20.04_x64", "raspbian9"]
+    if args.os in os_x86_64_list:
+        build_system = "linux-x86_64"
+    zip_filename=f'couchbase-lite-c-{args.edition}-{args.version}-{args.bld_num}-{build_system}.tar.gz'
```

Comment:
build_system will not be defined if the if condition is not met.

Maybe add: 

build_system = {args.os}

Before the if?